code-security/admin_guide/scan-monitor/custom-build-policies/custom-build-policies.adoc

```diff
@@ -2,6 +2,11 @@
 
 Prisma Cloud includes out-of-the-box policies that enable you to detect misconfigurations and provide automated fixes for security issues seen across your integrated code repositories and pipelines. You can review this list of Configuration policies with a filter for subtype *Build* on the Prisma Cloud administrative console *Policies*.
 
+You can create custom build policies for the following formats:
+
+* *Terraform* - Policies written using Terraform attributes will apply for Terraform (.tf and plan files).
+* *CloudFormation* - Policies written using CloudFormation attributes will apply for CloudFormation, Serverless, and Cloud Deevelopment Kit (CDK).
```

Comment:
@tsmithv11 what do we mean by Serverless here?
serverless application model (SAM)
serverless applications and functions?

Reply:
AWS Serverless Application Model (SAM)